source/app/components/DocumentActionBar/DocumentActionBar.js

```diff
@@ -0,0 +1,60 @@
+import { Flex, Box, Button, HStack, useMediaQuery } from '@chakra-ui/react';
+import { useDispatch } from 'react-redux';
+import ExportPreview from './ExportPreview/ExportPreview';
+import { clearRedactions } from '../../store/entities/documents/actions';
+import DocumentSearchBar from '../DocumentSearchBar/DocumentSearchBar';
+import OrangeButton from '../Button/Button';
+import { ExclusionListMenu } from './ExclusionListMenu/ExclusionListMenu';
+import { getAreRedactionsOnDocument } from '../../utils/document';
+
+const DocumentActionBar = ({ document, isComplianceTrack, redactMatches }) => {
+  const dispatch = useDispatch();
+  const { documentId } = document;
+  const [showRedactionLegend] = useMediaQuery('(min-width: 1550px)');
+
+  return (
+    <Flex py={3} px='2.96875vw' justifyContent='space-between' alignItems='center'>
+      {isComplianceTrack && showRedactionLegend && (
+        <Box
+          aria-hidden
+          pointerEvents='none'
+          boxShadow='0 0 0 2px #000'
+          bg='rgba(128, 128, 128, 0.2)'
```

Comment:
For colors, should we stick to one model of color picking between RGBA and HEX?
Could use RGBA all across or (personal preference) HEX + alpha

Reply:
You're not able to set alpha channels using hex

Reply:
I think you can add alpha to Hex codes. Have a look at this [this](https://www.digitalocean.com/community/tutorials/css-hex-code-colors-alpha-values#adding-an-alpha-value-to-your-hex-codes)

Reply:
Ah it's relatively new but seems [well supported](https://caniuse.com/css-rrggbbaa)

Reply:
My primary concern is not around choosing rgba over hex but more around one color model throughout for consistency. 
However, if you strongly about keeping both, I can work with that

Reply:
I agree that we should use one model when possible as well - just trying to balance it out with the fact that percentages represented as hexadecimal feel wrong.

Worth noting that this is purely a maintenance consideration and not something related to business logic, as opposed to if we were mixing HSL with RGB which is lossy conversion.